### PR TITLE
Add the logging utility and log the error into log file, console

### PR DESCRIPTION
**Pull request by Celthi**
@takashiro 

This PR adds an logging utility to log the error, debug, or info level information.
Error messages are still logged into console when debugging.

This PR also fix a minor issue "Syntax error in JSON parse" in the test when running with --local-debug.

Could you help to review and see if you could merge it :).

Thanks

### CODE REVIEW

File: core/App.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ class App {
 	 * @return {Promise}
 	 */
 	start() {
+		logger.info(`Starting the server in port ${this.config.socket}.`)
```
Comment (takashiro) [inline core/App.js]: Unix socket may be used as this.config.socket. e.g. "/var/run/asmodee-werewolf.sock". The log message is a bit misleading sometimes. How about `Starting the server at ${this.config.socket}.`?

File: .vscode/launch.json [hunk 2]
```diff
@@ -15,7 +43,9 @@
...
 			"name": "Launch Test",
 			"program": "${workspaceFolder}/test.js",
-			"args": ["--local-debug"]
+			"args": [
+				"--local-debug"
+			]
 		}
 	]
-}
+}
```
Comment (takashiro) [inline .vscode/launch.json]: A new line is needed to eliminate the warning here. Ref: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

File: config/log4js.dev.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            "level": "info"
+        },
+        "asmodee-server": {
+            "appenders": [
+                "logToFile", "stdout", "console"
+            ],
+            "level": "info"
+        }
+    }
+}
```
Comment (takashiro) [inline config/log4js.dev.json]: New line here as well.

File: config/log4js.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            "level": "error"
+        },
+        "asmodee-server": {
+            "appenders": [
+                "logToFile", "stdout"
+            ],
+            "level": "info"
+        }
+    }
+}
```
Comment (takashiro) [inline config/log4js.json]: New line here.

**Comment by Celthi** [comment]
Checking with ESLint done.

File: test/units/index.js [hunk 5]
```diff
@@ -7,4 +7,5 @@ module.exports = [
 	require('./fetchroles'),
 	require('./roomcapacity'),
 	require('./thiefcard'),
+	require('./mongoTest')
```
Comment (takashiro) [inline test/units/index.js]: CI failed here. The source file doesn't exist.

Comment (Celthi) [reply test/units/index.js]: The functionality for recording user info is not ready, so I missed checking this file.
Correct with new commit.

**Pull request merged**